packages/playwright/src/reporter.ts

```diff
@@ -340,22 +350,37 @@ export default class ReplayPlaywrightReporter implements Reporter {
   async onEnd() {
     try {
       await this.reporter.onEnd();
-      if (!this._foundReplayBrowser) {
-        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-browser");
 
-        const output: string[] = [];
+      const output: string[] = [];
+
+      const executedProjectWithReplay = !Object.keys(this._projects).some(projectName => {
+        const { executed, usingReplay } = this._projects[projectName];
+        return executed && usingReplay;
+      });
+
+      if (!executedProjectWithReplay) {
+        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-project");
         output.push(emphasize("None of the configured projects ran using Replay Chromium."));
-        if (!existsSync(getRuntimePath())) {
+      }
+
+      if (executedProjectWithReplay && !existsSync(getRuntimePath())) {
+        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-browser");
+        if (output.length) {
           output.push("");
-          output.push(`Install Replay Chromium by running ${highlight("npx replayio install")}`);
         }
+        output.push(`Install Replay Chromium by running ${highlight("npx replayio install")}`);
```

Comment:
Should we add a message before this too so it's less floaty. Something that gives context like "To record tests with Replay, you need to install the Replay browser"?

Reply:
Something like that? I like the idea that we keep this in a single line
```suggestion
        output.push(`To record tests with Replay, you need to install the Replay browser ${highlight("npx replayio install")}`);
```